src/main/java/com/serhiiklymchuk/gmailpet/config/WebConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.serhiiklymchuk.gmailpet.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebConfig implements WebMvcConfigurer {
+
+    @Bean
+    public BCryptPasswordEncoder getPasswordEncoder(){
+        return new BCryptPasswordEncoder();
+    }
+
+    @Override
+    public void addViewControllers(ViewControllerRegistry registry) {
+        registry.addRedirectViewController("/", "/login");
```

Comment:
you haven't fix the comment

Reply:
It will redirect to `/inbox` in the next PR, it's here only because of demonstrational purposes

Reply:
Actually, it's not recommended to implement something "just for future", that it out of the scope of your task (as long as this redirect is not in the scope of the task).
But let it be for now.

Reply:
Actually, it's not recommended to implement something "just for future", that it out of the scope of your task (as long as this redirect is not in the scope of the task).
But let it be for now.